Sources/GetStartedApp/Configuration.swift

```diff
@@ -33,6 +33,18 @@ public class ConfigManager {
                }
                return CloudantCredentials(url: url, username: username, password: password)
            }
+        }
+
+        // This is to account for cases where a Cloudant service broker is not available,
+        // and bound credentials must be provided to the application by 'user provided services' 
+        if let service = services["user-provided"] as? [Any] {
+           if let d = service[0] as? [String: Any], let credentials = d["credentials"] as? [String: Any] {
```

Comment:
Up to you, but we could check the whole array

Reply:
I have reached out to Elena to see if there are any test cases where this is needed.  I will make this change later if needed, but what is here is sufficient for the user-provided service values test case.